assets/sass/components/footer.scss

```diff
@@ -0,0 +1,22 @@
+.footer {
+  position: absolute;
+  bottom: 0px;
+  left: 0px;
+  right: 0px;
+  background-color: $color-primary;
+  padding: 20px;
+  display: flex;
+
+  p {
+    flex-grow: 1;
+    color: white;
+    margin: 0;
+    a {
+      color: white;
+    }
+  }
+
+  &__language-select {
+    float: right;
```

Comment:
As the container is a flex-container, you probably want to set `justify-content: space-between` on `.footer` instead of floating. It’s usually either flex or floating, not both.